modal/sandbox.py

```diff
@@ -91,7 +91,44 @@ async def _get_logs():
         return data
 
 
+class _StreamWriter:
+    """Provides an interface to buffer and write logs to a sandbox stream (`stdin`)."""
+
+    def __init__(self, sandbox_id: str, client: _Client):
+        self._index = 1
+        self._sandbox_id = sandbox_id
+        self._client = client
+        self._is_closed = False
+        self._buffer = bytearray()
```

Comment:
Probably want a max size check somewhere to prevent clients from filling their process memory indefinitely with unflushed buffer

Reply:
I was originally going to leave the flushing mechanism for the upcoming PR. But let me put together a basic implementation.